client/components/shipping/packages/add-package.js

```diff
@@ -10,170 +10,151 @@ import Dialog from 'components/dialog';
+			<FormFieldset>
+				<FormLabel>Inner Dimensions (L x W x H)</FormLabel>
+				<FormTextInput
+					name="inner_dimensions"
+					placeholder="100 x 25 x 5.5"
+					value={ inner_dimensions }
+					onChange={ ( event ) => updateFormTextField( event, updatePackagesField ) }
+				/>
+				{ renderOuterDimensionsToggle( showOuterDimensions, packageData, toggleOuterDimensions ) }
+			</FormFieldset>
+			{ renderOuterDimensions( showOuterDimensions, packageData, outer_dimensions, updatePackagesField ) }
+			<FormFieldset className="wcc-shipping-add-package-weight-group">
+				<div className="wcc-shipping-add-package-weight">
+					<FormLabel htmlFor="box_weight">Package weight</FormLabel>
 					<FormTextInput
-						id="name"
-						name="name"
-						placeholder="The customer will see this during checkout"
-						value={ name }
-						onChange={ this.updateFormTextField }
+						id="box_weight"
+						name="box_weight"
+						placeholder="Weight of box"
+						value={ box_weight }
+						onChange={ ( event ) => updateFormTextField( event, updatePackagesField ) }
 					/>
-				</FormFieldset>
-				<FormFieldset>
-					<FormLabel>Inner Dimensions (L x W x H)</FormLabel>
+				</div>
+				<div className="wcc-shipping-add-package-weight">
+					<FormLabel htmlFor="max_weight">Max weight</FormLabel>
 					<FormTextInput
-						name="inner_dimensions"
-						placeholder="100 x 25 x 5.5"
-						value={ inner_dimensions }
-						onChange={ this.updateFormTextField }
+						id="max_weight"
+						name="max_weight"
+						placeholder="Max weight"
+						value={ max_weight }
+						onChange={ ( event ) => updateFormTextField( event, updatePackagesField ) }
 					/>
-					{ this.renderOuterDimensionsToggle() }
-				</FormFieldset>
-				{ this.renderOuterDimensions( outer_dimensions ) }
-				<FormFieldset className="wcc-shipping-add-package-weight-group">
-					<div className="wcc-shipping-add-package-weight">
-						<FormLabel htmlFor="box_weight">Package weight</FormLabel>
-						<FormTextInput
-							id="box_weight"
-							name="box_weight"
-							placeholder="Weight of box"
-							value={ box_weight }
-							onChange={ this.updateFormTextField }
-						/>
-					</div>
-					<div className="wcc-shipping-add-package-weight">
-						<FormLabel htmlFor="max_weight">Max weight</FormLabel>
-						<FormTextInput
-							id="max_weight"
-							name="max_weight"
-							placeholder="Max weight"
-							value={ max_weight }
-							onChange={ this.updateFormTextField }
-						/>
-						<span className="wcc-shipping-add-package-weight-unit">{ weightUnit }</span>
-					</div>
-					<FormSettingExplanation> Define both the weight of the empty box and the max weight it can hold</FormSettingExplanation>
-				</FormFieldset>
-			</Dialog>
-		);
-	}
-}
+					<span className="wcc-shipping-add-package-weight-unit">{ weightUnit }</span>
+				</div>
+				<FormSettingExplanation> Define both the weight of the empty box and the max weight it can hold</FormSettingExplanation>
```

Comment:
Maybe say "Defines" instead of "Define".  Also there is a space ahead of Define that probably needs to go